sky/data/storage.py

```diff
@@ -437,15 +457,19 @@ def __init__(self,
         self.persistent = persistent
         self.mode = mode
         assert mode in StorageMode
+        self.interval_seconds = interval_seconds
```

Comment:
`interval_seconds` can be `None` when using `CSYNC` mode, which in case is set to `600`. Added `assert interval_seconds is None` when non-CSYNC mode is used.